process-migration/modules/process-migration/module.yaml

```diff
@@ -18,7 +18,7 @@ envs:
   value: "7.6.0"
 - name: "ADD_ONS_DISTRIBUTION_ZIP"
   value: "add_ons_distribution.zip"
-- name: "PROCESS_MIGRATION_DISTRIBUTION_JAR"
+- name: "KIE_PROCESS_MIGRATION_DISTRIBUTION_JAR"
```

Comment:
You renamed this variable to have a "KIE_" prefix here, but line 41 still has it without the prefix:
/opt/rhpam-process-migration/${PROCESS_MIGRATION_DISTRIBUTION_JAR}

Reply:
Also please change it in the build-overrides.sh script here:
https://github.com/jboss-container-images/jboss-kie-modules/blob/master/tools/build-overrides/build-overrides.sh#L763
and here:
https://github.com/jboss-container-images/jboss-kie-modules/blob/master/tools/build-overrides/build-overrides.sh#L781
I'm curious why this variable had to be changed in the first place?

Reply:
Totally right! Fixing it